### DISCUSSION TITLE
Probably a typo in API Reference for APIRouter.trace

### DISCUSSION THREAD

**Discussion by sepsh** [fastapi/fastapi#10493] [Questions] [answered]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
# Typo in Docs, no code here :)
```

### Description

Hello there
I was just reading the [APIRouter.trace refrence docs example](https://fastapi.tiangolo.com/reference/apirouter/#fastapi.APIRouter.trace--example)
and noticed there is a potential typo in the example (or I'm wrong, idk)

The typo is: the example is supposed to show `@router.trace` but it was written as `router.put`
the source code is [here](https://github.com/tiangolo/fastapi/blob/808e3bb9d56ac85e28e6f5e982f06209ad059436/fastapi/routing.py#L4296)

I'm not sure.
Also not so good in English :)

### Operating System

Linux

### Operating System Details

"Ubuntu 22.04.3 LTS (Jammy Jellyfish)"

### FastAPI Version

0.103.2

### Pydantic Version

2.4.2

### Python Version

Python 3.11.6

### Additional Context

The fix would be like the code-block below. Current source code at [here](https://github.com/tiangolo/fastapi/blob/808e3bb9d56ac85e28e6f5e982f06209ad059436/fastapi/routing.py#L4296)
```diff
from fastapi import APIRouter, FastAPI
        from pydantic import BaseModel

        class Item(BaseModel):
            name: str
            description: str | None = None

        app = FastAPI()
        router = APIRouter()

-        @router.put("/items/{item_id}")
+        @router.trace("/items/{item_id}")
        def trace_item(item_id: str):
            return None

        app.include_router(router)
```

**Comment by Kludex** [comment, marked as answer]
PR welcome. Ping me there, please.
